### ISSUE TITLE
[Bug]: Can not show information about transformation in the UI

### ISSUE THREAD

**Issue by samuell** [featureform/featureform#448]
### Expected Behavior

I expect to see some metadata information about the transformation when clicking on it.

### Actual Behavior

I only see the animated three dots, as if data is loaded:

![image](https://user-images.githubusercontent.com/125003/189502933-16bc1caf-c08a-4a40-ac63-5319d60cdaeb.png)

### Steps To Reproduce

Add this code to a file named `definitions.py`:

```python
import featureform as ff

ff.register_user("myself").make_default_owner()

local = ff.register_local()

dummydata = local.register_file(
    name="dummydata",
    variant="default",
    description="",
    path="data.csv",
)

person = ff.register_entity("person")

dummydata.register_resources(
    entity="person",
    entity_column="person_id",
    inference_store=local,
    features=[
        {
            "name": "foo",
            "variant": "default",
            "column": "foo",
            "type": "float32",
        },
        {
            "name": "bar",
            "variant": "default",
            "column": "bar",
            "type": "float32",
        },
    ],
    timestamp_column="time",
)


@local.df_transformation(variant="default", inputs=[("dummydata", "default")])
def compute_fooplusbar(df):
    df["fooplusbar"] = df["foo"] + df["bar"]
    return df


compute_fooplusbar.register_resources(
    entity=person,
    entity_column="person_id",
    inference_store=local,
    features=[
        {
            "name": "fooplusbar",
            "variant": "default",
            "column": "fooplusbar",
            "type": "float32",
        },
    ],
    labels=[
        {
            "name": "fooplusbar",
            "variant": "default",
            "column": "fooplusbar",
            "type": "float32",
        },
    ],
    timestamp_column="time",
)
```

Run:

```bash
featureform apply --local definitions.py
featureform dash
```

Enter the URL http://127.0.0.1:3000/sources/compute_fooplusbar

... or alternatively, open http://127.0.0.1:3000 and click "features" and then "compute_foobar" on the "source" row in the table with metadata.

### What mode are you running Featureform in?

Local

### What version of Python are you running?

3.9

### Featureform Python Package Version

1.1.12

**Comment by Sami1309** [comment]
https://github.com/featureform/featureform/pull/469

**Comment by Sami1309** [comment]
Issue resolved on updated python package 12.3
 https://pypi.org/project/featureform/
